Web Inspector: Adding Lizette to contributors.json

liz4662 opened the pull request:
#### fcc4154a89cdda633d002d2ea940171219f1ea30
<pre>
Web Inspector: Adding Lizette to contributors.json
<a href="https://bugs.webkit.org/show_bug.cgi?id=293391">https://bugs.webkit.org/show_bug.cgi?id=293391</a>
<a href="https://rdar.apple.com/151810077">rdar://151810077</a>

Reviewed by BJ Burg.

Added name and information to contributors list.

* metadata/contributors.json:

Canonical link: <a href="https://commits.webkit.org/295752@main">https://commits.webkit.org/295752@main</a>
</pre>

Ahmad-S792 commented on metadata/contributors.json:
```diff
@@ -5446,6 +5446,15 @@
+      ],
+      "github" : "liz4662",
+      "name" : "Lizette Hernandez",
+      "status" : "contributer"
```
I think default state itself is `contributor`, so you don't need to add this line.

Ahmad-S792 commented on metadata/contributors.json:
```diff
@@ -5446,6 +5446,15 @@
       ]
    },
+   {
+      "emails" : [
```
Plus you need to align this alphabetically and as per error, it should be above `Lorenzo Tilve` as below:

```
    {
       "emails" : [
         "lhernandez28@apple.com",
         "lizette4662@gmail.com"
      ],
      "github" : "liz4662",
      "name" : "Lizette Hernandez"
   },
```

liz4662 committed:
Web Inspector: Adding Lizette to contributors.json
https://bugs.webkit.org/show_bug.cgi?id=293391
rdar://151810077

Reviewed by BJ Burg.

Added name and information to contributors list.

* metadata/contributors.json:

Canonical link: https://commits.webkit.org/295752@main
